spanner/api/Spanner.Samples/CreateDatabaseWithMultiRegionEncryptionAsync.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2024 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_create_database_with_MR_CMEK]
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+using System.Threading.Tasks;
+
+public class CreateDatabaseWithMultiRegionEncryptionAsyncSample
+{
+    public async Task<Database> CreateDatabaseWithMultiRegionEncryptionAsync(string projectId, string instanceId, string databaseId, CryptoKeyName[] kmsKeyNames)
+    {
+        // Create a DatabaseAdminClient instance that can be used to execute a
+        // CreateDatabaseRequest with custom encryption configuration options.
+        DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
+        // Define create table statement for table #1.
+        var createSingersTable =
+        @"CREATE TABLE Singers (
```

Comment:
Nit: indent this line? (Ditto for createAlbumsTable)

Reply:
Done here and in all the create database samples.